package.json

```diff
@@ -25,7 +25,8 @@
     "standard-wdio": "wdio run ./tests/e2e/standard/wdio.conf.js",
     "default-wdio-mobile": "wdio run ./tests/e2e/default-mobile/wdio.conf.js",
     "local-images": "export VERSION=$(node ./scripts/build/get-version.js) && grunt build-service-images && grunt create-local-docker-compose-files",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "fresh-start": "npm ci && cd webapp && ng cache clean && echo \"The fresh-start process has completed!\""
```

Comment:
Please add `cd ../` so you end up in the same path.

Reply:
Thanks! Done